### ISSUE TITLE
Leverage `SUBTITLE` `NamedStyleType` to render different markdown

### ISSUE THREAD

**Issue by justinsunho** [xuopled/gatsby-source-google-docs#26]
`SUBTITLE` style types in our google docs weren't rendering onto our gatsby site. So our team customized the code to leverage the `SUBTITLE` NamedStyleType to map to `blockquote` since that type wasn't being used in this plugin.

`SUBTITLE`s should have a mapping so the text doesn't disappear, but we would like to see what you think it should map to. Any thoughts?

**Comment by justinsunho** [comment]
This is what we did
```
function getParagraphTag(p) {
  const tags = {
    NORMAL_TEXT: "p",
    SUBTITLE: "blockquote",
    HEADING_1: "h1",
    HEADING_2: "h2",
    HEADING_3: "h3",
    HEADING_4: "h4",
    HEADING_5: "h5",
  }

  return tags[p.paragraphStyle.namedStyleType]
}
```

**Comment by xuopled** [comment]
Thank you. Fixed in `v1.11.0`
